### PR TITLE
Deploy an alpha version only if PR has `deploy-alpha` label

### PR DESCRIPTION
**Pull request by VitaliyBoyko**
**Description** (*)
This PR adds the possibility to deploy to alpha only needed changes to avoid a mess in the repository.

### CODE REVIEW

File: .github/workflows/gradlepublishalpha.yml [hunk 1]
```diff
@@ -23,6 +23,7 @@ jobs:
       - name: Grant execute permission for gradlew
         run: chmod +x gradlew
       - name: Run publish plugin
+        if: contains(github.event.pull_request.labels.*.name, 'deploy-alpha')
```
Comment (coderimus) [inline .github/workflows/gradlepublishalpha.yml:26]: 👍

Comment (VitaliyBoyko) [reply .github/workflows/gradlepublishalpha.yml:26]: Hope it gonna work 🙏

Comment (coderimus) [reply .github/workflows/gradlepublishalpha.yml:26]: Looks correct according to this: https://help.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#example-using-an-array

Comment (VitaliyBoyko) [reply .github/workflows/gradlepublishalpha.yml:26]: I'm not sure whether PR is accessible in the push event.

Comment (VitaliyBoyko) [reply .github/workflows/gradlepublishalpha.yml:26]: Let's see

**Pull request merged**